e2e/tests/interchain_accounts/base_test.go

```diff
@@ -234,7 +222,7 @@ func (s *InterchainAccountsTestSuite) TestMsgSendTx_FailedTransfer_InsufficientF
 	})
 }
 
-func (s *InterchainAccountsTestSuite) TestMsgSubmitTx_SuccessfulTransfer_AfterReopeningICA() {
+func (s *InterchainAccountsTestSuite) TestMsgSendTx_SuccessfulTransfer_AfterReopeningICA() {
```

Comment:
I renamed this test since `MsgSubmit` is used in intermix, but in ibc-go we use `MsgSend`.